client/js/components/user/CustomerSettings/CustomerSettingsMap.vue

```diff
@@ -104,30 +119,68 @@ export default {
 
   data () {
     return {
-      currentMapAttribution: this.mapAttribution,
-      layer: this.initLayer,
-      layerUrl: this.initLayerUrl,
+      mapAttribution: this.initMapAttribution,
+      baseLayerLayers: this.initLayer,
+      baseLayerUrl: this.initLayerUrl,
       mapKey: 0
     }
   },
 
   computed: {
+    ...mapState('customer', {
+      customerItems: 'items'
+    }),
+
     attributionControl () {
       return new Attribution({ collapsible: false })
     }
   },
 
   methods: {
+    ...mapActions('customer', {
+      saveCustomer: 'save'
+    }),
+
+    ...mapMutations('customer', {
+      updateCustomer: 'setItem'
+    }),
+
     debounceUpdate: debounce(({ id, value }) => {
       if (id === 'r_baseLayerUrl' || id === 'r_baseLayerLayers') {
         if (id === 'r_baseLayerUrl') {
-          this.layerUrl = value
+          this.baseLayerUrl = value
         } else if (id === 'r_baseLayerLayers') {
-          this.layer = value
+          this.baseLayerLayers = value
         }
         this.mapKey = Math.floor(Math.random() * 1000)
       }
-    }, 1000)
+    }, 1000),
+
+    resetMapSettings () {
+      const lastState = this.customerItems[this.currentCustomerId].attributes
+
+      this.mapAttribution = lastState.mapAttribution
+      this.baseLayerLayers = lastState.baseLayerLayers
+      this.baseLayerUrl = lastState.baseLayerUrl
+    },
+
+    saveMapSettings () {
+      console.log('saveMapSettings')
```

Comment:
please remove the` console.log()`

Reply:
done here https://github.com/demos-europe/demosplan-core/pull/2212/commits/7b6ce1f81656a91c4012b3bcff690b124e511104